### ISSUE TITLE
TTL field doesn't unmarshal to the appropriate time

### ISSUE THREAD

**Issue by sdemjanenko** [sensedeep/dynamodb-onetable#396]
When model field is marked with `ttl: true`, it accepts a date object and is stored as a value in epoch seconds in Dynamo since Dynamo TTL fields require the value to be in seconds. When this is then loaded back out of dynamo, it initializes a Date object, but it doesn't get multiplied by 1000, leaving the date to be in the year 1970.

**Comment by mobsense** [comment]
Can you post a reproduction as per the template for us to reproduce and check it out.

Thanks.

**Comment by mobsense** [comment]
TransformReadAttribute has code to handle this. So it should work.

A test case is needed:
```
    transformReadAttribute(field, name, value, params, properties) {
        if (typeof params.transform == 'function') {
            //  Invoke custom data transform after reading
            return params.transform(this, 'read', name, value, properties)
        }
        if (field.type == 'date' && value != undefined) {
            if (field.ttl) {
                return new Date(value * 1000)
            } else {
                return new Date(value)
            }
        }
```

**Comment by sdemjanenko** [comment]
Ah, it looks like this was fixed in v2.5. I'm running 2.3.8 presently.
